src/main/java/com/esri/samples/na/offline_routing/README.md

```diff
@@ -10,7 +10,7 @@
 adding at least 2 stops, a route will display. Use the combo box in the top left corner to choose between the travel 
 modes "Fastest" and "Shortest" (how the route is optimized). To move a stop, right-click the graphic to select it, 
 then move your mouse to reposition, and finally right-click again to set the new position. The route will update 
-on-the-fly while moving stops.</p>
+on-the-fly while moving stops. The green box marks the boundary of the route geodatabase.</p>
```

Comment:
Don't know if you want to update the image to show the green outline box. When I read this I think, what green box?